apps/devhub/utils.py

```diff
@@ -54,6 +54,7 @@ def limit_validation_results(validation, is_compatibility=False):
             validation['messages'].append({
                 'tier': 1,
                 'type': msgtype,
+                'id': [],  # To respect the message structure, see bug 1139674.
```

Comment:
Mind giving this an id? I think there are normally 3 strings but I'm not sure if that's required, or if it just ends up that way. I'd try to stretch it to 3. `['validation', 'messages', 'truncated']` or something, which is terrible but matches more closely with the other `id`s.